slasher/cache/highest_attestation_cache.go

```diff
@@ -0,0 +1,62 @@
+package cache
+
+import (
+	lru "github.com/hashicorp/golang-lru"
+	slashpb "github.com/prysmaticlabs/prysm/proto/slashing"
+)
+
+var (
+	// validatorsCacheSize defines the max number of validators public keys the cache can hold.
+	highestAttCacheSize = 100000
```

Comment:
i think 300000 is better as we don't want to write and read from db

Reply:
@shayzluf done